src/main/resources/com/google/api/codegen/php/main.snip

```diff
@@ -332,9 +332,17 @@
      *                              that don't use retries. For calls that use retries,
      *                              set the timeout in RetryOptions.
      *                              Default: 30000 (30 seconds)
-     *     @@type string $appName The codename of the calling service. Default 'gax'.
-     *     @@type string $appVersion The version of the calling service.
-     *                              Default: the current version of GAX.
+     *     @@type string $libName The API library software used for calling the service.
+     *                            (Unless you are writing an API client itself, leave this as default.)
+     *     @@type string $libVersion The API library software version used for calling the service.
+     *                               (Unless you are writing an API client itself,
+     *                               leave this as default.)
+     *     @@type string $appName Currently unsupported - reserved for future use.
+     *                            The name of the application calling the service. Recommended
+     *                            for analytics purposes.
+     *     @@type string $appVersion Currently unsupported - reserved for future use.
```

Comment:
Luke is a big fan of avoiding breaking changes wherever possible, so that's why this was done in Python (it's a cultural thing for that language community). Also, we will have a future use case for `$appName/$appVersion`, but we don't want to do anything with these values right now. I'm leaving that for a future design round.